scheduler/scheduler.go

```diff
@@ -1091,6 +1091,109 @@ func (driver *MesosSchedulerDriver) abort(cause error) (stat mesos.Status, err e
 	return
 }
 
+func (driver *MesosSchedulerDriver) AcceptOffers(offerIds []*mesos.OfferID, operations []*mesos.Offer_Operation, filters *mesos.Filters) (mesos.Status, error) {
+	driver.eventLock.Lock()
+	defer driver.eventLock.Unlock()
+
+	if stat := driver.status; stat != mesos.Status_DRIVER_RUNNING {
+		return stat, fmt.Errorf("Unable to AcceptOffers, expected driver status %s, but got %s", mesos.Status_DRIVER_RUNNING, stat)
+	}
+
+	if !driver.connected {
```

Comment:
actually you probably want to send TASK_LOST to the Scheduler for each LAUNCH op. see launchTasks() and `pushLostTask`. the native code does this: https://github.com/apache/mesos/blob/master/src/sched/sched.cpp#L1160